test-server.js

```diff
@@ -1,24 +1,27 @@
 const request = require('supertest');
 const expect = require('chai').expect;
 
-
-describe('test server', function () {
+describe('test server', () => {
     let server;
 
-    beforeEach(function () {
+    beforeEach(() => {
```

Comment:
Was this change intentional?
If not, please revert. Lambdas function slightly differently to traditional `function()` as they take the context of `this` and lose the Mocha context. While the test currently work fine - this may cause headaches in future tests